commands/curation/curationaudit.go

```diff
@@ -1002,6 +1009,40 @@ func getMavenNameScopeAndVersion(id, artiUrl, repo string, node *xrayUtils.Graph
 	}
 	return downloadUrls, strings.Join(allParts[:2], ":"), "", allParts[2]
 }
+func getGradleNameScopeAndVersion(id, artiUrl, repo string, node *xrayUtils.GraphNode) (downloadUrls []string, name, scope, version string) {
```

Comment:
please add comments to this func (inside and outside the describe it), simmilar to `getMavenNameScopeAndVersion`